src/options/flags.rs

```diff
@@ -1,95 +1,383 @@
 use crate::options::parser::{Arg, Args, TakesValue, Values};
 
 // exa options
-pub static VERSION: Arg = Arg { short: Some(b'v'), long: "version",  takes_value: TakesValue::Forbidden };
-pub static HELP:    Arg = Arg { short: Some(b'?'), long: "help",     takes_value: TakesValue::Forbidden };
+pub static VERSION: Arg = Arg {
```

Comment:
I see that your editor, like mine, doesn't seem to respect the `.rustfmt.toml` file that says to skip this file. I had to use regular vim to edit this file because neovim wouldn't stop formatting it on save

Reply:
I was wondering how it was a 500+ line change XD

Reply:
Oddly enough, I have rustfmt disabled in my neovim config for formatting on save, so I'm not sure how or why this happened! It should be fixed in the coming commits.